# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../lua/lua-5.1.4-pkgconfig-install-modules-path.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From cb8667a63c9d9544e0043b183133f3b5a6d7fe40 Mon Sep 17 00:00:00 2001
From: Christian Wiese <chris@opensde.org>
Date: Tue, 24 Apr 2012 14:55:36 +0200
Subject: [PATCH] lua.pc.in: add INSTALL_{L,C}MOD variables

Note:

Most distributions are defining INSTALL_LMOD and INSTALL_CMOD within their
packaged pkg-config file of lua, because both are commonly used by third-party
Makefiles for figuring out where to install lua modules through pkg-config.

INSTALL_LMOD defines the path where to install pure lua modules
INSTALL_CMOD defines the path where to install compiled lua modules
---
 etc/lua.pc.in |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/etc/lua.pc.in b/etc/lua.pc.in
index 2c4ef36..b01622d 100644
--- a/etc/lua.pc.in
+++ b/etc/lua.pc.in
@@ -5,6 +5,9 @@ exec_prefix=${prefix}
 libdir= @libdir@
 includedir=${prefix}/include
 
+INSTALL_LMOD= ${prefix}/share/lua/${V}
+INSTALL_CMOD= ${prefix}/lib/lua/${V}
+
 Name: Lua
 Description: An Extensible Extension Language
 Version: ${R}
-- 
1.7.2.3

